conans/test/integration/command/new_test.py

```diff
@@ -45,6 +45,20 @@ def test_new_missing_definitions(self):
         client.run("new cmake_lib -d version=myversion", assert_error=True)
         assert "Missing definitions for the template. Required definitions are: 'name', 'version'" in client.out
 
+    def test_new_basic_template(self):
+        tc = TestClient()
+        try:
+            tc.run("new basic")
+            assert '# self.requires("zlib/1.2.13")' in tc.load("conanfile.py")
+        except Exception as e:
+            assert False, f"conan new basic should work without any extra arguments: {e}"
```

Comment:
```suggestion
    tc.run("new basic")
    assert '# self.requires("zlib/1.2.13")' in tc.load("conanfile.py")
```
It is not necessary to capture unexpected exceptions. If that happens the test would fail anyway. Otherwise we would need to capture absolutely every ``tc.run()`` with try-except just for an extra message? 
In other words, line 54 is never executed, which doesn't make much sense.